src/components/text/marked/highlight/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+
+import { MarkedTextProps } from '..';
+
+import Underline from './Underline';
+import Highlight from './Highlight';
+
+export type HighlightProps = Pick<
+  MarkedTextProps,
+  'variant' | 'highlightColor'
+>;
+
+const MarkedTextHighlight: FC<HighlightProps> = (props) => {
```

Comment:
Why did you call the component Highlight if it switches between Highlight and Underline?

Reply:
The whole thing is a multipart component composed of `container`, `text` and `highlight` (because it looks as if highlighter is used to mark the text), what if we rename it to `mark`?

Reply:
Sounds reasonable